webapp/storybook/helpers.js

```diff
@@ -57,6 +60,16 @@ const helpers = {
       </layout>`,
     }
   },
+  fakeUser(n) {
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-04-07T14:39:06Z" title="Tuesday, April 7th 2020, 4:39:06 pm +02:00">Apr 7, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -57,6 +60,16 @@ const helpers = {
       </layout>`,
     }
   },
+  fakeUser(n) {
```


very cool helper! 😍

Reply:
> [<img alt="rbeer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/rbeer) **Authored by [rbeer](https://github.com/rbeer)**
_<time datetime="2020-04-16T09:27:50Z" title="Thursday, April 16th 2020, 11:27:50 am +02:00">Apr 16, 2020</time>_

---


Thanks :)
The only thing I don't like is that I'm using it again, in the tests.
https://github.com/Human-Connection/Human-Connection/pull/3293/files/1ac34a8fbadccacbdf92b074eff1caa02d59afeb#diff-970486bf70217f927b18aadea0c8b1b4R4